src/rust/ide/enso-protocol/README.md

```diff
@@ -0,0 +1,91 @@
+Enso protocol consists of pair server and client. They communicate by exchanging
+messages, following JSON-RPC 2.0.
```

Comment:
This file should be not here, but in docs instead. And for me it looks outdated (because it looks like it was written for FileManager service designed by Michał, but then it became obsolete. And the protocol description is enso repository, so I think we can remove this file entirely.